src/components/auth/Login.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import Login from './Login';
+
+test('allows the user to login successfully', async () => {
```

Comment:
There are a few suggestions about this test:

1 - Group a set of common tests inside a `describe` function. [Example here](https://github.com/phoenixdo/internship-job-portal.git).

2 - There are no expectations on this test. You're firing the behavior but you are not checking if the results of the behavior makes sense. Example: If you click in the login button, you need to expect that the login logic will be triggered. [Example here](https://jestjs.io/docs/en/using-matchers).

3 - There are no bad-path tests. You assumed the user will always do the good UI path here instead of assuming that s/he can make mistakes. Example: If the user is missing the password the login button function should not be called. You also can test if the messages `Password is required` are rendering. [Example here](https://testing-library.com/docs/react-testing-library/cheatsheet)

4 - For testing the login logic (when you implement it) the best approach is to extract the login into a external function that can be mocked and have it calls expected. [Example here](https://jestjs.io/docs/en/mock-functions)